src/routes/+layout.svelte

```diff
@@ -23,6 +23,11 @@
 	import { staleTime } from '$data/api';
 	import RouterEvents from '$data/RouterEvents';
 
+	// @ts-ignore
+	import { DateTriggerEventsHandler, DateTriggerEvent } from 'datetrigger';
+	// @ts-ignore
+	import { DEFAULT_EVENTS } from 'datetrigger/presets';
```

Comment:
why the ts-ignores? does your library not include typedefs?

Reply:
nope, plain js, the repo is linked in the pr description

Reply:
adding ts annotations would be appreciated

Reply:
then I'm gonna draft this pr and learn typescript

Reply:
you mentioned to move the code in a script tag, is typescript needed in this case?

Reply:
well in that case, you end up having to manually bundle the code and insert it as a string, so correct

Reply:
done in latest version of the lib